### ISSUE TITLE
Auto Encoder Viz - plot distribution

### ISSUE THREAD

**Issue by zrm22** [maize-genetics/MLImpute#117]
Plot out the distributions of the logits split between inputs of 1 and 0.
